tests/queries/0_stateless/02011_dictionary_empty_attribute_list.sql

```diff
@@ -0,0 +1,12 @@
+DROP TABLE IF EXISTS test_table;
+CREATE TABLE test_table (id UInt64) ENGINE=TinyLog;
+INSERT INTO test_table VALUES (0);
+
+DROP DICTIONARY IF EXISTS test_dictionary;
+CREATE DICTIONARY test_dictionary (id UInt64) PRIMARY KEY id LAYOUT(DIRECT()) SOURCE(CLICKHOUSE(TABLE 'test_table'));
+SELECT * FROM test_dictionary;
+SELECT dictHas('test_dictionary', toUInt64(0));
+SELECT dictHas('test_dictionary', toUInt64(1));
```

Comment:
Maybe add `dictGet` with some dummy name. Just to be sure that nothing crashed before attributed validation.